doc/source/management/innodb_expanded_fast_index_creation.rst

```diff
@@ -4,11 +4,7 @@
 |feature|
 ================================================================================
 
-:Availability:
-
-   The |feature| feature is not available in |Percona Server| |release|. This
-   feature will be included into a later release.
-
+:Availability: In version 8.0.17-8, this feature is **Experimental** qualtiy.
```

Comment:
But this arrived to 8.0 much earlier, ask @georgelorch @georgelorchpercona which version to put here

Reply:
EFI was introduced in 8.0.15-5 https://jira.percona.com/browse/PS-4892 and is at best considered experimental.

Reply:
Good. Updated.